app/policies/application_policy.rb

```diff
@@ -57,7 +57,32 @@ def initialize(user, scope)
     end
 
     def resolve
+      return scope.all unless user.rbac_enabled?
```

Comment:
```
  def resolve
    return graphql_id_query if graphql_query_by_id

    return scope.all unless user.rbac_enabled?

    resolve_scope
  end
```

Reply:
`graphql_id_query`  depends on rbac check. it will fail if rbac is disabled.

Reply:
but you cannot return `scope.all` if the query is for one id

Reply:
`resolve` is always called by `index` to return a list. Even for `graphql_query_by_id`, it returns an association with single instance.